binder/src/main/java/io/grpc/binder/SecurityPolicies.java

```diff
@@ -42,18 +42,21 @@
 
 /** Static factory methods for creating standard security policies. */
 @CheckReturnValue
-@ExperimentalApi("https://github.com/grpc/grpc-java/issues/8022")
 public final class SecurityPolicies {
 
   private static final int MY_UID = Process.myUid();
   private static final int SHA_256_BYTES_LENGTH = 32;
 
   private SecurityPolicies() {}
 
+  @ExperimentalApi("https://github.com/grpc/grpc-java/issues/8022")
   public static ServerSecurityPolicy serverInternalOnly() {
     return new ServerSecurityPolicy();
   }
 
+  /**
+   * Creates a default {@link SecurityPolicy} that checks authorization based on UID.
```

Comment:
"... that allows access only to callers with the same UID as the current process."